llvm/lib/Target/RISCV/MCTargetDesc/RISCVInstPrinter.cpp

```diff
@@ -34,6 +34,10 @@ static cl::opt<bool>
               cl::desc("Disable the emission of assembler pseudo instructions"),
               cl::init(false), cl::Hidden);
 
+static cl::opt<bool> EmitX8AsFP("riscv-emit-x8-as-fp",
```

Comment:
Can we move this to a `-M` option, rather than a hidden `cl`? I think we're trying to move all uses of `riscv-no-aliases` to the equivalent `-M` option.